shared/js/initApp.js

```diff
@@ -13,7 +13,8 @@ window.apps = {
   load: loadApp,
   // Legacy Blockly initialization that was moved here from _blockly.html.haml.
   // Modifies `appOptions` with some default values in `baseOptions`.
-  setup: function () {
+  // Move blockly-specific setup function out of shared and back into dashboard.
+  setupBlockly: function () {
```

Comment:
Nit: this is a bit of a misnomer since these options are also used to set up Droplet levels.